### ISSUE TITLE
Tests for twisted.scripts should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7727]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7727|
|Type|enhancement|
|Created|2014-11-28 11:23:13Z|
|Branch|https://github.com/twisted/twisted/tree/scripts-testcase-names-7727|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7727.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b50/b50b5f131f24502131814c98636d9e78dabb9be6/c65ca6a15b89daca9a744a88115d1bfb64156a49.patch) (1265 bytes) - added by eeshangarg on 2014-11-28 11:25:39Z - 
* [bugfix_7727_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b50/b50b5f131f24502131814c98636d9e78dabb9be6/cbda407a13d2d1cb91c59c20e81d08b3f293c6d4.patch) (584 bytes) - added by eeshangarg on 2015-04-22 13:12:44Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7727 7727
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_scripts_testcase_names_7727 branches/scripts-testcase-names-7727
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1417173793029417 1417173793029417
changetime__1429864330149537 1429864330149537
version__None None
owner__eeshangarg eeshangarg
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7727.patch​) that changes all of the tests in twisted.scripts to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [43753]) Branching to scripts-testcase-names-7727.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @eeshangarg|
|-|-|

Thanks for working on this.
Changes looks good.

While touching this code you have also awaken the twistedchecker beast.

Since the testcase was changes, it looks like twistedchecker is confused and can not recognize that the errors already exists in previous code.

If you have time, please take a look at current errors and try to fix them.
They should be a good exercise in working with apidoc and documenting functions using the epydoc formatting.

See the list of errors here:

https://buildbot.twistedmatrix.com/builders/twistedchecker/builds/2488/steps/run-twistedchecker/logs/new%20twistedchecker%20errors

In case you are lost, just let me know and I can help or I will fix this.

If posssible, please send a new patch on top of branches/scripts-testcase-names-7727

When you have a question and see me on IRC don't hesitate to ask.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_4"></a>|@eeshangarg removed owner|
|-|-|

Thanks for the review!

I just added a new patch called bugfix_7727_2.patch​. Tests for tap2rpm and tap2deb should not be touched as tap2rpm and tap2deb have been deprecated in Twisted 15.2. Therefore, we can ignore the twistedchecker warnings, since they are about documenting the tests for tap2rpm. My latest patch does not include changes made to the tests for tap2rpm/tap2deb in my previous patch. :-)

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @eeshangarg|
|-|-|

Thanks!
Sorry for my initial comment which created a lot of wasted work.

Changes sent to buildbot and will merge once all test are ok.... and they should :)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

(In [44545]) Merge scripts-testcase-names-7727 : Update twisted.scripts test to the new style of naming test classes.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7727](https://github.com/twisted/twisted/issues/7727)
